packages/babel-node/test/index.js

```diff
@@ -73,15 +65,9 @@ const assertTest = function(stdout, stderr, opts) {
 
   if (opts.stdout) {
     if (opts.stdoutContains) {
-      assert.ok(
-        includes(stdout, expectStdout),
-        "stdout " +
-          JSON.stringify(stdout) +
-          " didn't contain " +
-          JSON.stringify(expectStdout),
-      );
+      expect(includes(stdout, expectStdout)).toBeTruthy();
```

Comment:
Can we somehow keep the old error message?

Reply:
`expect(stdout).stringContaining(expectStdout)`

Reply:
I don't know if `stringContaining` will work out of the box here, since I believe we are using `lodash.includes` because stdout is a String in some cases and an object in some.